index.html

```diff
@@ -24,8 +24,127 @@ <h1>I’m Mariane Glad to see you!</h1>
+            <div class="card1">
+            <h2>Tonic</h2>
+            <ul> 
+                <li>CANOPY</li>
+                <li>⬤</li>
+                <li>BackEnd Dev</li>
+                <li>⬤</li>
+                <li>2015</li>
+            </ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+            </ul>
+            <button>See Project</button>
+        </div>
+        <div class="cards card2">
+            <img src="./assets/Snapshoot1.png" alt="">
+            <div class="card1">
+                <h2>Multi-Post Stories</h2>
+                <ul> 
+                    <li>CANOPY</li>
+                    <li>⬤</li>
+                    <li>BackEnd Dev</li>
+                    <li>⬤</li>
+                    <li>2015</li>
+                </ul>
+                </div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+            <button>See Project</button>
+            </div>
+        <div class="cards card3">
+            <img src="./assets/Snapshoot2.png" alt="">
+            <div class="card1">
+                <h2>Tonic</h2>
+                <ul> 
+                    <li>CANOPY</li>
+                    <li>⬤</li>
+                    <li>BackEnd Dev</li>
+                    <li>⬤</li>
+                    <li>2015</li>
+                </ul>
+                </div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+            <button>See Project</button>
+            </div>
+        <div class="cards card4">
+            <img src="./assets/Snapshoot3.png" alt="">
+            <div class="card1">
+                <h2>Multi-Post Stories</h2>
+                <ul> 
+                    <li>CANOPY</li>
+                    <li>⬤</li>
+                    <li>BackEnd Dev</li>
+                    <li>⬤</li>
+                    <li>2015</li>
+                </ul>
+                </div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+    </section>
+    <section class="about">
+        <div class="about-social">
+        <h2>About Myself</h2>
```

Comment:
- [ ] Please increase the font weight for about section heading and make sure your design matches with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/185661751-dbaf5200-1675-40ba-a674-9b8a50713a80.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/185661659-f0240662-d269-469a-be61-b283bceaf507.png)